### PR TITLE
adding visualization of dataset

### PR DESCRIPTION
**Pull request by lucifertrj**
Updating `Image Captioning.py` and `Image Captioning.ipynb` by adding visualization of the Image Data.

```py
def plot_images_data(filename):
    """
    Loads the image data 
    params: filename: File path that contains the image data
    """
    #define the size of plot window
    plt.figure(figsize=(20,20))
    for view_images in range(9):
        plt.subplot(3,3,view_images+1)
        #resize and plot the image
        img = cv2.imread(f"{filename}/{random.choice(os.listdir(filename))}")
        img = cv2.resize(img,(200,200),interpolation=cv2.INTER_CUBIC)
        plt.axis("off")
        plt.imshow(img)
```

### CODE REVIEW

File: examples/vision/image_captioning.py [hunk 1]
```diff
@@ -69,7 +71,22 @@
 """
 ## Preparing the dataset
 """
-
+def plot_images_data(filename):
+    """
```
Comment (fchollet) [inline examples/vision/image_captioning.py:75]: Please use the standard format of Keras docstrings.

**Review by fchollet — COMMENTED** [review]
Thanks for the PR!

**Review by lucifertrj — COMMENTED** [review]
I am fixed the docstrings @fchollet

**Comment by lucifertrj** [comment]
Hey @fchollet, 

Should anything else need to be updated here?

**Comment by lucifertrj** [comment]
@fchollet, any updates?

**Comment by pcoet** [comment]
Hi @lucifertrj, thanks for the PR. I believe @fchollet wants you to follow [Google style for doc strings](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#381-docstrings). So this

```
def plot_images_data(filename):
    """
    Loads the image data 
    params: filename: File path that contains the image data
    """
```

would look like this

```
def plot_images_data(filename):
    """Loads the image data.

    Args:
        filename: File path that contains the image data.
    """
```

Could you 1) fix the merge issue; 2) update the docstring; 3) regenerate the notebook and markdown?

**Pull request closed without merging**